src/screenshots/covidcast/map_preview.py

```diff
@@ -0,0 +1,127 @@
+"""Takes a screenshot of the COVIDcast map.
+
+Extraneous UI is hidden so that the map is presentable in standalone form. The
+screenshot is intended to be suitable for use in social media previews and
+custom dashboards.
+"""
+
+# standard library
+import argparse
+import time
+import urllib.parse
+
+# third party
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium import webdriver
+
+# first party
+from delphi.operations.screenshots.utils import Utils
+
+
+class MapPreview:
+
+  # URL of the COVIDcast map with predetermined signals
+  URL = 'https://covidcast.cmu.edu/?' + urllib.parse.urlencode({
+    'sensor': 'doctor-visits-smoothed_adj_cli',
+    'level': 'county',
+    'date': '20200918',
+    'signalType': 'value',
+    'encoding': 'color',
+    'mode': 'overview',
+    'region': '42003',
+  })
+
+  # width and height in pixels of the final screenshot
+  SCREENSHOT_SIZE = (1800, 900)
+
+  # width and height in pixels of the virtual screen needed to capture a
+  # screenshot of the desired size (e.g. to allow room for surrounding UI
+  # elements, window borders, etc)
+  SCREEN_SIZE = (SCREENSHOT_SIZE[0], SCREENSHOT_SIZE[1] + 311)
+
+  # location of the center of Allegheny county in pixels relative to the
+  # top-left corner of the map
+  ALLEGHENY_COUNTY_LOCATION = (1077, 372)
```

Comment:
minor, but maybe insert PIXEL in the name to distinguish from e.g., latlng.

Reply:
removed this constant in favor of command-line args. added pixel terminology there in the help string.